src/main/java/net/neoforged/neoforge/common/IOUtilities.java

```diff
@@ -33,9 +38,15 @@ public final class IOUtilities {
             StandardOpenOption.WRITE,
             StandardOpenOption.TRUNCATE_EXISTING
     };
+    private static final ThreadFactory THREAD_FACTORY = new NamedThreadFactory("Neoforge-IOUtilities");
```

Comment:
Does this factory set tbe correct context ClassLoader?

Reply:
For what reason is that necessary? The docs for `getContextClassloader` says 
> If not set, the default is to inherit the context class loader from the parent thread.